index.js

```diff
@@ -3,7 +3,7 @@ const fs = require('node:fs');
 const path = require('node:path');
 const { Client, Collection, Events, GatewayIntentBits } = require('discord.js');
 const { token } = require('./config.json');
-const axios = require('axios');
+//const axios = require('axios');
```

Comment:
You can just delete this if we're not using it

Reply:
removed!